src/Stratis.Bitcoin.IntegrationTests/TestRegTest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Stratis.Bitcoin.Networks;
+using Stratis.Bitcoin.Tests.Common;
+
+namespace Stratis.Bitcoin.IntegrationTests
+{
+    /// <summary>
+    /// This network enables an immutable Consensus object for testing only.
+    /// </summary>
+    internal class TestRegTest : BitcoinRegTest
```

Comment:
Will we need a TestStratisRegTest as well?

Reply:
Possibly. There are other fields on Consensus that still have setters used in other tests.